misk/src/test/kotlin/misk/web/jetty/WebActionsServletTest.kt

```diff
@@ -0,0 +1,123 @@
+package misk.web.jetty
+
+import com.netflix.concurrency.limits.limit.SettableLimit
+import com.netflix.concurrency.limits.limiter.SimpleLimiter
+import misk.Action
+import misk.ApplicationInterceptor
+import misk.asAction
+import misk.client.UnixDomainSocketFactory
+import misk.inject.KAbstractModule
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import misk.time.FakeClock
+import misk.time.FakeClockModule
+import misk.web.*
```

Comment:
Don't use `*` importts

Reply:
typically I would agree with this but I would make exceptions for tests